src/FunctionApps/python/GenerateCSVs/elr.py

```diff
@@ -1,8 +1,39 @@
 from typing import List, Any
-from GenerateCSVs.patient import PATIENT_COLUMNS
+from GenerateCSVs.patient import PATIENT_COLUMNS, parse_patient_resource
 
-ELR_COLUMNS = PATIENT_COLUMNS + []
+
+ELR_COLUMNS = PATIENT_COLUMNS + ["loinc_code", "result", "effective_date"]
+
+
+def extract_covid_test(observation: dict) -> dict:
```

Comment:
I'm not seeing any code here to actually identify tests are covid-related.  I think that's in line with what we decided on Slack, but should we update the function name (and adjust some test cases) so it doesn't seem like we are trying to filter to only covid results?

Reply:
Yeah that's fair, it's really just filtering for lab results that are test/result pairs. I can update the semantics.